app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ActiveRecord::Base
+  belongs_to :users, class_name: 'User', optional: true
+  belongs_to :posts, class_name: 'Post', optional: true
```

Comment:
- [REQUIRED] Consider removing the optional flags because every comment should have an author and should also belong to a post
- [REQUIRED] for the users we can have an author that will also be a user.
- Therefore the relationships above can be expressed as below
```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```